examples/dev/trace-analytics-sample-app/opentelemetry-collector/otel-collector-config.yml

```diff
@@ -7,7 +7,8 @@ receivers:
 exporters:
   otlp/2:
     endpoint: data-prepper:21890
-    insecure: true
+    insecure: false
+    insecure_skip_verify: true
```

Comment:
QUES: how is it different from `insecure: true`?

Reply:
* insecure: toggles TLS on/off
* insecure_skip_verify: if using TLS,  toggles whether cert signing should be enforced

Full documentation: https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md#tls--mtls-configuration

Reply:
I see. I misunderstood TLS only as verifying the certificates.